packages/wallet-sdk/src/createCoinbaseWalletSDK.ts

```diff
@@ -13,11 +14,11 @@ import { assertPresence } from ':util/assertPresence.js';
 import { checkCrossOriginOpenerPolicy } from ':util/checkCrossOriginOpenerPolicy.js';
 import { validatePreferences, validateSubAccount } from ':util/validatePreferences.js';
 import { createCoinbaseWalletProvider } from './createCoinbaseWalletProvider.js';
-import { SubAccount, ToSubAccountSigner, store } from './store/store.js';
+import { SubAccount, ToOwnerAccountFn, store } from './store/store.js';
 
 export type CreateCoinbaseWalletSDKOptions = Partial<AppMetadata> & {
   preference?: Preference;
-  toSubAccountSigner?: ToSubAccountSigner;
+  subAccounts?: SubAccountOptions;
```

Comment:
I think this top level should just be subAccount? Any reason to make it plural?

Reply:
that's the way it was in the spec. no strong preference here, will update

Reply:
Sorry about that. What do you feel is better. I'm leaning towards singular but no strong preference

Reply:
hm... i suppose there's an argument to be made for `subAccounts` plural because from the developer pov it applies to the sub accounts of all users. i suggest we leave it as is for now and see how it feels - still time to change if we change our minds